pkg/service/constants/constant.go

```diff
@@ -4,24 +4,10 @@ package constants
 //calling StrPtr() or aws.String() seems like repetetive
 
 var (
-	NameLabel           = "Name" //Capital N for Aws
-	CreatorLabel        = "creator"
-	SpawnerServiceLabel = "spawner-service"
-	Scope               = "scope"
-
-	ProvisionerLabel         = "provisioner"
-	ClusterNameLabel         = "cluster-name"
-	WorkspaceLabel           = "workspaceid"
-	NodeNameLabel            = "node-name"
-	InstanceLabel            = "instance"
-	NodeLabelSelectorLabel   = "nodeLabelSelector"
-	AwsLabel                 = "aws"
-	VpcTagKey                = "vpc"
-	NBTypeTagkey             = "nb-type"
-	NBRegionWkspNetworkStack = "nb-region-ntwk-stk"
-	WorkspaceId              = "workspaceid"
-	AzureLabel               = "azure"
-	GcpLabel                 = "gcp"
+	WorkspaceId = "workspaceid"
```

Comment:
We can remove this as it's not used anymore.

Reply:
used in cost API implementation. so keeping it as is now